posts/2022-05-19-Rust-1.61.0.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: "Announcing Rust 1.61.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.61.0. Rust is a programming language
+empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via rustup, you can get 1.61.0 with:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install]
+from the appropriate page on our website, and check out the
+[detailed release notes for 1.61.0][notes] on GitHub.
+
+If you'd like to help us out by testing future releases, you might consider updating locally to use
+the beta channel (`rustup default beta`) or the nightly channel (`rustup default nightly`).
+Please [report] any bugs you might come across!
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1610-2022-05-19
+[report]: https://github.com/rust-lang/rust/issues/new/choose
+
+## What's in 1.61.0 stable
+
+### Custom exit codes from `main`
+
+In the beginning, Rust `main` functions could only return the unit `()`, always indicating success
```

Comment:
```suggestion
In the beginning, Rust `main` functions could only return the implicit unit type `()`, always indicating success
```

Reply:
nit: this phrasing seems a bit misleading to me. The unit type isn't implicit; it seems more like it was implicit that it was the return type of the function. This also seems a bit clunky. How about `could only implicitly return" or "always implicitly returned"? (not entirely happy with my phrasing either, just wordsmithing)

Reply:
Really, it could be either implicit or explicit, `fn main() {}` or `fn main() -> () {}`, while you `return`, `return ()`, or just end the block.

Reply:
That's true, but I feel like saying *just* "could only return the unit `()`" may slightly confuse people who don't regularly think about the implicit `-> ()`.

Edited further for clarity.